webapp/components/ContributionForm/ContributionForm.spec.js

```diff
@@ -384,6 +392,15 @@ describe('ContributionForm.vue', () => {
           await wrapper.find('form').trigger('submit')
           expect(mocks.$apollo.mutate).toHaveBeenCalledWith(expect.objectContaining(expectedParams))
         })
+
+        it('deletes a teaser image', async () => {
+          expectedParams.variables.image = null
```

Comment:
Could you please hard-code the entire `expectedParams` here?

Reply:
as we discussed in our pairing session, I understand the motive for this, but `expectedParams` is quite big, and is only 10 lines of code above this, so I think it's ok like this.